apps/simple-trading-app/src/app/components/header/logo.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { t } from '@vegaprotocol/react-helpers';
+
+const Logo = () => {
+  return (
+    <NavLink aria-label={t('Go to home page')} to="/">
+      <svg
```

Comment:
I assume that this is Vega "V". Can we make component from this?

Reply:
done